components/editor/modules/embed/index.js

```diff
@@ -76,6 +77,17 @@ export const getTwitterEmbed = gql`
   }
 `
 
+export const getCommentEmbed = gql`
+  query getCommentEmbed($id: ID!) {
+    comments(focusId: $id) {
+      focus {
+        id
+        text
+      }
+    }
+  }
+`
+
```

Comment:
@patrickvenetz the other embed queries have a matching `embed` gql queries, whereas here I used the `comments` query. Please advise.

Reply:
Not sure about implications just yet.

But gutfeeling is to stick to your approach with `comments` API.